src/presentation/http/router/note.test.ts

```diff
@@ -501,10 +537,7 @@ describe('Note API', () => {
 
     test('Returns 406 when the id does not exist', async () => {
       /** Create test user */
-      const user = await global.db.insertUser({
-        email: 'test1@codexmail.com',
-        name: 'CodeX1',
-      });
+      const user = await global.db.insertUser();
 
       const nonexistentId = 'ishvm5qH84';
```

Comment:
is that variable still needed?

Reply:
i think if we remove it, there would be just magic number, so yes